### PR TITLE
[BBCSPA-1494] warningModal component

### PR DESCRIPTION
**Pull request by Tuxosaurus**
Normal (mobile)
warningmodal

Large (desktop)
warningmodal_large

### CODE REVIEW

File: src/_utils/branding.ts [hunk 1]
```diff
@@ -49,6 +49,10 @@ export const font: Font = {
     size: '30px',
     lineHeight: String(Math.floor((32 / 30) * 100) / 100),
   },
+  xxl: {
```
Comment (pith) [inline src/_utils/branding.ts]: `xxxxl` coming soon 😉

Comment (Tuxosaurus) [reply src/_utils/branding.ts]: One Letter Per Screen™

**Comment by PerrineBoissieres** [comment]
I'd do a refactoring with the Modal component, but probably in another PR ^^

File: src/_utils/branding.ts (hunk 1 above)
Comment (Zarthia) [reply src/_utils/branding.ts]: xxl is the biggest one

**Comment by Tip-Sy** [comment]
Looks good to me!

**Pull request merged**